contracts/src/tokens/migration/TokenMigration.sol

```diff
@@ -28,29 +30,34 @@ contract TokenMigrationFacet is
     IERC20 oldToken,
     IERC20 newToken
   ) external onlyInitializing {
+    _validateTokens(oldToken, newToken);
+
     TokenMigrationStorage.Layout storage ds = TokenMigrationStorage.layout();
     (ds.oldToken, ds.newToken) = (oldToken, newToken);
     _pause();
   }
 
   /// @inheritdoc ITokenMigration
-  function migrate(address account) external whenNotPaused {
+  function migrate(address account) external whenNotPaused nonReentrant {
```

Comment:
There's no arbitrary external call causing reentrancy.

Reply:
fixed in latest push